playbooks/roles/learner-profile-dashboard/tasks/deploy-app.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: checkout repository
+  git:
+    repo: "{{ LPD_REPOSITORY }}"
+    version: "{{ LPD_REPOSITORY_VERSION }}"
+    dest: "{{ LPD_APP_LOCATION }}"
+
+- name: install dependencies
+  pip:
+    virtualenv: "{{ LPD_VENV }}"
+    state: present
+    requirements: "{{ LPD_APP_LOCATION }}/requirements/base.txt"
+    virtualenv_python: "/usr/bin/python2.7"
+
+# This is a hack Gunicorn==17.5 is installed system wide, so virtualenv and system version match.
+# possibly this should be sloved in a cleaner way
+- name: install dependencies
+  pip:
+    virtualenv: "{{ LPD_VENV }}"
+    state: present
+    name: "Gunicorn==19.4.5"
+    virtualenv_python: "/usr/bin/python2.7"
+
+- name: install dependencies
```

Comment:
@tomaszgy Change to `install mysql` (or `install mysql client`) to be more specific?